### PR TITLE
3652 Remove STFPd account.url support.

### PR DESCRIPTION
**Pull request by jaraco**
Alternative to #1020, remove `accounts.url` support in SFTPd.

Per [#3652](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3652).

### CODE REVIEW

**Comment by jaraco** [comment]
Tests seemed to fail for an unrelated reason. I've rebased the changes against the latest master. Let's see how it fares now.

**Comment by itamarst** [comment]
Removing this is a policy question someone from core committers should weigh in on.

**Comment by exarkun** [comment]
> Removing this is a policy question someone from core committers should weigh in on.

I am :+1: on this removal (presumably not surprise, at least to jaraco, since I suggested it to him :smile: ).

We traced the history of this feature to an allmydata.com-provided account-management service.  Clearly that offering is defunct (and has been for over a decade) and the source was never made public as far as I know.  There is nothing that would prevent someone else from building an implementation of the server-side component of this (and the protocol is pretty simple) but I'm not aware that anyone ever has.

**Review by exarkun — APPROVED** [review]

**Pull request merged**
